chapter_did/chapter_did.Rmd

```diff
@@ -541,116 +773,152 @@ grid.brackets(x,
 
-$\mathbb{E}(EMP_{st} \; | \; TREAT_s = 1, POST_t = 1) = \alpha + \beta + \gamma + \delta$
+$$\mathbb{E}[Y_{ist}^1 - Y_{ist}^0|s,t] = \delta$$
 
---
+Outcomes in the treatment group:
 
-$$[\mathbb{E}(EMP_{st} \; | \; TREAT_s = 1, POST_t = 1)-\mathbb{E}(EMP_{st} \; | \; TREAT_s = 1, POST_t = 0)] - \\
-[\mathbb{E}(EMP_{st} \; | \; TREAT_s = 0, POST_t = 1)-\mathbb{E}(EMP_{st} \; | \; TREAT_s = 0, POST_t = 0)] = \delta$$
+$$\mathbb{E}[Y_{ist}|s = \text{New Jersey}, t = \text{Feb}] = \gamma_{NJ} + \lambda_{Feb}$$
+
+$$\mathbb{E}[Y_{ist}|s = \text{New Jersey}, t = \text{Nov}] = \gamma_{NJ} + \delta + \lambda_{Nov}$$
 
+$$
+\begin{align}
+\mathbb{E}[Y_{ist}&|s = \text{New Jersey}, t = \text{Nov}] - \mathbb{E}[Y_{ist}|s = \text{New Jersey}, t = \text{Feb}] \\
+&= \gamma_{NJ} + \delta + \lambda_{Nov} - (\gamma_{NJ} + \lambda_{Feb}) \\
+&= \delta + \lambda_{Nov} - \lambda_{Feb}
+\end{align}
+$$
 
 ---
 
-class: inverse
+# Parallel Trends Assumption: Formally
 
-# Task 2 (10 minutes)
+Let $\delta$ denote the true impact of the minimum wage increase:
 
-* Create a dummy variable, `treat`, equal to 0 if `state` is Pennsylvania and 1 if New Jersey.
+$$\mathbb{E}[Y_{ist}^1 - Y_{ist}^0|s,t] = \delta$$
 
-* Create a dummy variable, `post`, equal to 0 if `observation` is February 1992 and 1 otherwise.
+Outcomes in the treatment group:
 
-* Estimate the following regression model using the `lm` function:
+$$\mathbb{E}[Y_{ist}|s = \text{New Jersey}, t = \text{Feb}] = \gamma_{NJ} + \lambda_{Feb}$$
 
-$$empfte_{st} = \alpha + \beta treat_s + \gamma post_t + \delta_{DiD} (treat_s \times post_t) + \varepsilon_{st}$$
+$$\mathbb{E}[Y_{ist}|s = \text{New Jersey}, t = \text{Nov}] = \gamma_{NJ} + \delta + \lambda_{Nov}$$
 
-* Do you obtain the same results as before?
+$$
+\begin{align}
+\mathbb{E}[Y_{ist}&|s = \text{New Jersey}, t = \text{Nov}] - \mathbb{E}[Y_{ist}|s = \text{New Jersey}, t = \text{Feb}] \\
+&= \gamma_{NJ} + \delta + \lambda_{Nov} - (\gamma_{NJ} + \lambda_{Feb}) \\
+&= \delta + \underbrace{\lambda_{Nov} - \lambda_{Feb}}_{\text{time trend}}
+\end{align}
+$$
 
 ---
 
-# Multi-period DiD: Relaxing Common Trends Assumption => Too much?
+# Parallel Trends Assumption: Formally
 
-* When we have data on many years and many states we can relax the common trends assumption
+Therefore we have:
 
-$$EMP_{st} = \alpha + \delta_{DiD} MW_st + \sum_{k=Alaska}^{Wyoming} \beta_k STATE_{kt} + \sum_{j=1985}^{1999} \gamma_j YEAR_{jt} + \varepsilon_{st}$$
+$$
+\begin{align}
+\mathbb{E}[Y_{ist}&|s = \text{PA},t = \text{Nov}] - \mathbb{E}[Y_{ist}| s = \text{PA},t = \text{Feb}] = \underbrace{\lambda_{Nov} - \lambda_{Feb}}_{\text{time trend}}
+\end{align}
+$$
 
----
+--
 
-# Clustered Standard Errors => Too much?
+$$
+\begin{align}
+\mathbb{E}[Y_{ist}&|s = \text{NJ},t = \text{Nov}] - \mathbb{E}[Y_{ist}| s = \text{NJ},t = \text{Feb}] = \delta + \underbrace{\lambda_{Nov} - \lambda_{Feb}}_{\text{time trend}}
+\end{align}
+$$
 
+--
 
+$$
```

Comment:
i think this discussion is very good. take good time to explain that slowly. it's sufficiently easy to be understable I would say.